ci/container/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs_driver_test",
-  "version": "1.6.18",
+  "version": "1.6.19",
```

Comment:
If upcoming release is 1.6.20 then it should be 1.6.20.

Reply:
looks to be already submitted with #444